### PR TITLE
PortAudio Library Upgraded to Latest Stable Release, v19.7.0.

### PR DESCRIPTION
**Pull request by pgcrowe**
This pull upgrades the PortAudio Library to the latest stable release, v19.7.0. [http://files.portaudio.com/download.html](http://files.portaudio.com/download.html) This upgrade fixes the PortAudio driver errors occurring in GrandOrgue on macOS 11. **NOTE: Build was successful on macOS 11. However, testing is needed on Linux and Windows before the commit is approved. If any others developers would be so kind as to test on those platforms, I would appreciate it.**

### CODE REVIEW

**Comment by pgcrowe** [comment]
@oleg68: My apologies for my delay on updates to this commit. I've been inundated with work this past week. I'll try to continue debugging later this week. In the meantime, feel free to make changes as you see fit.

File: src/portaudio/CMakeLists.txt [hunk 1]
```diff
@@ -17,31 +17,31 @@ endif()
...
-  common/pa_converters.c
-  common/pa_cpuload.c
-  common/pa_debugprint.c
-  common/pa_dither.c
-  common/pa_front.c
-  common/pa_process.c
-  common/pa_ringbuffer.c
-  common/pa_stream.c
-  common/pa_trace.c)
+  src/common/pa_allocation.c
```
[the remark this answers is not in the document]
Comment (pgcrowe) [reply src/portaudio/CMakeLists.txt:20]: I tried {CMAKE_SOURCE_DIR} a few commits ago, and it didn't work for some reason. I'll try again, I may have been lacking the $ or something.

File: .gitmodules [hunk 2]
```diff
@@ -5,3 +5,6 @@
 	path = submodules/RtMidi
 	url = https://github.com/thestk/rtmidi
 	branch = 4.0.0
+[submodule "submodules/PortAudio"]
+	path = submodules/PortAudio
+	url = https://github.com/PortAudio/portaudio.git
```
Comment (oleg68) [inline .gitmodules:10]: Please fix the submodule to the latest release tag

**Review by oleg68 — CHANGES_REQUESTED** [review]
It stiil needs to add subdirectory src/portaudio instead of submodules

**Comment by oleg68** [comment]
@pgcrowe I've pushed a PR https://github.com/pgcrowe/grandorgue/pull/1 to your fork with suggested changes.

**Review by oleg68 — CHANGES_REQUESTED** [review]
Everything looks OK, but resolve the conflicts, please

**Review by oleg68 — CHANGES_REQUESTED** [review]
Resolving conflicts is required

**Comment by oleg68** [comment]
@pgcrowe resolving conflicts in configurations.xml is hard, so just take the new version from upstream.

**Comment by oleg68** [comment]
@pgcrowe Because you haven't fix your PR, I've created another one with your commits: #845

**Pull request closed without merging**